### PR TITLE
Add Vagrant support.

### PR DESCRIPTION
**Pull request by wkordalski**
1. Moved role-related directories to `roles/socorro` - I think it is more intuitive, readable and does not depends on name of directory and maybe contents of parent directory. If somebody want to use these scripts, they can copy or symlink a directory.

2. Added Vagrant box configuration to easily experiment with Ansible configuration.

### CODE REVIEW

File: ansible.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [defaults]
-roles_path=../
+roles_path=roles/
```
Comment (ruslo) [inline ansible.cfg]: Currently code fits recommended directory structure:
* https://galaxy.ansible.com/intro#share

I think this part should not be changed.

**Review by ruslo — CHANGES_REQUESTED** [review]

File: tasks/main.yml [hunk 2]
```diff
@@ -247,28 +233,52 @@
...
   become: false
 
 - debug: msg="{{ x.cmd }}"
+  when: inventory_hostname!='vagrant'
 - debug: msg="{{ x.stdout_lines }}"
+  when: inventory_hostname!='vagrant'
 
-- debug: msg="Visit http://{{ inventory_hostname }}:5601 and search for 'socorro_reports'"
+- debug: msg="Visit http://{{ ansible_default_ipv4.address }}:5601 and search for 'socorro_reports'"
+  when: inventory_hostname!='vagrant'
```
Comment (ruslo) [inline tasks/main.yml:284]: I don't think relying of inventory file name is a good idea. This should be some condition variable

```
- debug: msg="..."
  when: ansible_socorro_use_vargant
```

File: prod.yml [hunk 3]
```diff
@@ -2,7 +2,17 @@
 # All Rights Reserved.
 
 ---
-- hosts: socorro
+
+- hosts: vagrant
```
Comment (ruslo) [inline prod.yml:6]: I don't know what exactly is happenning here but according to the code it looks like something quite orthogonal to socorro installation. Why it's not a separate role?

**Review by ruslo — CHANGES_REQUESTED** [review]
Please fix Travis CI testing.

**Comment by wkordalski** [comment]
According to the discussion with @ruslo, Vagrant files will be moved to separate repository.

**Pull request closed without merging**